gtsam/linear/NoiseModel.h

```diff
@@ -266,7 +266,20 @@ namespace gtsam {
       /// Compute covariance matrix
       virtual Matrix covariance() const;
 
-    private:
+      /**
+       * @brief Helper method to compute the normalization constant
+       * for a Gaussian noise model.
+       * k = 1/log(\sqrt(|2πΣ|))
```

Comment:
I thought we defined k as $1/(\sqrt(|2πΣ|)$. This should return $\log(k)$ (and say so).

Reply:
Oops sorry about that.